packages/docops/src/preview-front.ts

```
@@ -170,15 +172,15 @@ async function main() {
   const uuid = args["--uuid"] || undefined;
   const file = args["--file"] || undefined;
-  const res = await computePreview(
-    { uuid, file },
-    {
-      dir: ROOT,
-      docThreshold: DOC_THRESHOLD,
-      refThreshold: REF_THRESHOLD,
-      debug: DEBUG,
-    },
-  );
+  const frontSel: { uuid?: string; file?: string } = {};
```

ESLint

[error] Variable should have an immutability of at least "ReadonlyDeep" (actual: "Mutable").

(functional/prefer-immutable-types)